debian/rules

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/make -f
+include /usr/share/dpkg/pkg-info.mk
+export DH_VERBOSE = 1
+export PATH := $(shell printenv PATH):/usr/lib/go-1.24/bin
+export DEB_BUILD_MAINT_OPTIONS = optimize=-lto
+export GOTOOLCHAIN := local
+export INSIGHTS_GO_PACKAGE := $(shell grep-dctrl -s XS-Go-Import-Path -n - ./debian/control)
+export GOFLAGS := -ldflags=-X=$(INSIGHTS_GO_PACKAGE)/internal/constants.Version=$(DEB_VERSION_UPSTREAM) --mod=vendor -buildmode=pie
+export DH_GOLANG_BUILDPKG := github.com/ubuntu/ubuntu-insights/cmd/insights
+export DH_GOLANG_GO_GENERATE := 1
+
+BUILDDIR := _build
+
+%:
+	dh $@ --builddirectory=$(BUILDDIR) --buildsystem=golang
+
+override_dh_auto_clean:
+	dh_auto_clean
+
+	# Vendor Go dependencies when building the source package
+	[ -d vendor/ ] || go mod vendor
+
+override_dh_auto_install:
+	mv $(BUILDDIR)/bin/insights $(BUILDDIR)/bin/ubuntu-insights
+	mv $(BUILDDIR)/src/$(INSIGHTS_GO_PACKAGE)/generated $(CURDIR)/debian/tmp
+	dh_auto_install -- --no-source
```

Comment:
honest question: the fact to override this parameter does not prevent `--fail-missing` to still be active? (something to test, we want fail-missing).